src/login/pam_systemd.c

```diff
@@ -847,7 +847,6 @@ static uint64_t pick_default_capability_ambient_set(
 
 typedef struct SessionContext {
         const uid_t uid;
-        const pid_t pid;
```

Comment:
For the commit message:
> This recomes a weird asymmetry, as create_session_message() always went
to the process' own PID when ding pidfds but otherwise took the pid as
input.

I cannot parse that.